### PR TITLE
Update README.md and setup.py

### PR DESCRIPTION
**Pull request by zleung9**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,9 @@
     author="shobrook",
     author_email="shobrookj@gmail.com",
     # classifiers=[],
-    install_requires=[],
+    install_requires=[
+        "json_repair",
```
Comment (shobrook) [inline setup.py:29]: I think `"json-repair"` would be correct here. Also, could you please add `"openai"` and `"tiktoken"` here too?

File: README.md [hunk 2]
```diff
@@ -265,19 +265,19 @@ The `Evaluator` object has the following parameters:
...
+In most cases, simply customizing this object will be sufficient, but in some situations it makes sense to build your own evaluator. For example, if you're building a coding agent, you may want to evaluate a search trajectory using some external feedback, such as whether the code compiles or whether a set of unit tests are passing. To build a custom evaluator, you must extend the `Evaluator` base class and implement a `run` method. This method must take in a list of [Message](#the-message-object) objects as input, representing a search trajectory, and return an [`Evaluation` object](https://github.com/shobrook/saplings/blob/master/src/dtos/Evaluation.py) as output. This object has two properties: `score` (a value between 0 and 1) and `reasoning` (an _optional_ string with written justification for the score).
```
Comment (shobrook) [inline README.md:268]: Could you please fix the link here as well? I think `src` needs to be replaced with `saplings`. Probably true for other instances in the README as well.

**Review by shobrook — CHANGES_REQUESTED** [review]
Thanks for opening this! Happy to merge once you address my two quick comments.

**Comment by zleung9** [comment]
Not a problem. Just realized you addressed those requests yourself already. Thanks!

**Pull request merged**